code/modules/food_and_drinks/food/foods/candy.dm

```diff
@@ -94,6 +94,51 @@
 	filling_color = "#7D5F46"
 	goal_difficulty = FOOD_GOAL_NORMAL
 
+/obj/item/food/candy/chocolateorange
```

Comment:
All of these with combined words need to be converted to snake_case. It needs to follow our coding conventions for the sake of readability.

Reply:
Alright. Do you want the same done for icon paths too?

Reply:
Preferably. In this case, it helps to keep things readable. It also makes it consistent, because some of them are already in the proper format.

Reply:
Yes. In this case, it helps to keep things readable. It also makes it consistent, because some of them are already in the proper format

Reply:
Alright, will do. Some of them are only like that as I largely copied how they were in TG's files.